Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml

```diff
@@ -1406,6 +1406,23 @@
   - type: Sprite
     sprite: Objects/Consumable/Drinks/pwineglass.rsi
 
+- type: entity
+  parent: DrinkGlassBase
+  id: DrinkPoscaGlass
+  name: posca glass
+  description: Poor warriors' drink from a forgotten era.
+  components:
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 50
+        reagents:
+        - ReagentId: Posca
+          Quantity: 50
+  - type: Drink
+  - type: Sprite
+    sprite: Objects/Consumable/Drinks/glass_lyellow.rsi
```

Comment:
what is a lyellow

Reply:
new sprite with `l`ight `yellow` drink texture

Reply:
You can call it light_yellow

Reply:
understood, ill fix it later today

Reply:
Done